test/functional/rpc_blockchain.py

```diff
@@ -410,6 +411,13 @@ def _test_getblock(self):
         self.log.info("Test that getblock with verbosity 2 still works with pruned Undo data")
         datadir = get_datadir_path(self.options.tmpdir, 0)
 
+        try:
+            self.log.info("Test that getblock with invlaid verbosity type returns proper error message")
+            node.getblock(blockhash, "2")
```

Comment:
You can use the `assert_raises_rpc_error` helper

Reply:
Okie updated to use `assert_raises_rpc_error` instead.